src/main/java/mods/natura/items/StickItem.java

```diff
@@ -26,7 +25,6 @@ public StickItem() {
     @SideOnly(Side.CLIENT)
     @Override
     public IIcon getIconFromDamage(int meta) {
-        if (meta >= textureNames.length) return Items.leather_helmet.getIconFromDamage(meta);
```

Comment:
You can use `Util.getWithFallback` [`a716d2b` (#16)](https://github.com/GTNewHorizons/Natura/pull/16/commits/a716d2b9742040bb0194c152d7135c2c5324eff7)

Reply:
I dont see why I would want to give textures to stuff that doesnt exist. most mods dont.

Reply:
there always is the default fallback: the flame. and it clearly tells you that something is broken and needs fixing.

Reply:
Well in the case I linked it crashed the game. As long as it doesn't, sure